sfa_dash/static/js/form-validation.js

```diff
@@ -0,0 +1,45 @@
+MAX_FILESIZE = 16777216;
+$(document).ready(function(){
+    // Custom validity handlers to supply users with more meaningful messages
+    // when they provide invalid input.
+    if(document.getElementsByName('name').length >0){
+        // Listen for invalid input
+        document.getElementsByName('name')[0].addEventListener(
+            'invalid', function(){
+                this.setCustomValidity(
+                    `Name may only contain characters a-z, A-Z, 0-9,
```

Comment:
looks like we're missing comma, apostrophe, and space? Might be easier to understand if they are all together without the commas like ``a-zA-Z0-9(), -'_``